Seohee/Lv2/다리를 지나는 트럭.js

```diff
@@ -0,0 +1,33 @@
+function solution(bridge_length, weight, truck_weights) {
+    // 다리위의 트럭
+    let onBridge = [];
+    let onBridgeTime = [];
+    //총시간
+    let time = 1;
+    // 무게 합산
+    let sum = truck_weights[0];
+    //맨처음 트럭 가져오고 시간 초기화
+    onBridge.push(truck_weights.shift())
+    onBridgeTime.push(1);
```

Comment:
```suggestion
    const onBridge = [truck_weights.shift()];
    const onBridgeTime = [1];
    //총시간
    let time = 1;
    // 무게 합산
    let sum = onBridge[0];
```
이렇게 표현하면 2줄을 줄일 수 있어요.